website/docs/r/cloud_backup_schedule.html.markdown

```diff
@@ -64,67 +42,33 @@ resource "mongodbatlas_cloud_backup_schedule" "test" {
   reference_minute_of_hour = 45
   restore_window_days      = 4
 
-}
-```
-
-### Second step - Add 2 policies items
-```hcl
-resource "mongodbatlas_cluster" "my_cluster" {
-  project_id   = "<PROJECT-ID>"
-  name         = "clusterTest"
-  disk_size_gb = 5
-
-  //Provider Settings "block"
-  provider_name               = "AWS"
-  provider_region_name        = "EU_CENTRAL_1"
-  provider_instance_size_name = "M10"
-  cloud_backup     = true // must be enabled in order to use cloud_backup_schedule resource
-  provider_disk_iops          = 100
-}
-
-resource "mongodbatlas_cloud_backup_schedule" "test" {
-  project_id   = mongodbatlas_cluster.my_cluster.project_id
-  cluster_name = mongodbatlas_cluster.my_cluster.name
 
-  reference_hour_of_day    = 3
-  reference_minute_of_hour = 45
-  restore_window_days      = 4
-  
+  // This will now add the desired policy items to the existing mongodbatlas_cloud_backup_schedule resource
   policies {
     id = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.id
 
     policy_item {
-      frequency_interval = 2
+      id                 = mongodbatlas_cluster.my_cluster.snapshot_backup_policy.0.policies.0.policy_item.0.id
```

Comment:
I misunderstood, will rollback to TypeList for `policies` since with MaxItems 1 it behaves like a object, for  `policies_items` changed to TypeSet because if you change the order it should no detect diff similar to the issue INTMDB-202

Reply:
Okay, Got it on that but does the general comment here and in the PR response make sense?  @thetonymaster can you perhaps help out here?   See the comment above from yesterday.